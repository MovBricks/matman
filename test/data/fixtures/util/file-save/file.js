var json = {
    name: 'matman',
    age: 16
};

module.exports = json;