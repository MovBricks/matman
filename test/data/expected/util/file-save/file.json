{"name":"matman","age":16}
